### DISCUSSION TITLE
how to fetch data from different models with property/value parameters?

### DISCUSSION THREAD

**Discussion by NirinaTanjona** [prisma/prisma#12754] [Q&A] [answered]
Hi guys, I'd like to retrieve data in get request using ```{property}={value}``` pair as parameters. I don't know how to deal with it so I need your help.
this is what I want do implement: 
```
http://localhost:3000/api/users?gender=female&location.city=Paris
```
So in this get request, I want to get all female users who live in Paris. Since user and location are from different model, I don't know how can I use ```findMany``` and ```where```condition.
 this is how my schema looks like:
```prisma
model User {
  id         Int          @id @default(autoincrement())
  gender     String?      @db.VarChar(255)
  email      String?      @db.VarChar(255)
  phone      String?      @db.VarChar(255)
  cell       String?      @db.VarChar(255)
  nat        String?      @db.VarChar(255)
  dob        Dob[]
  location   Location[]
  login      Login[]
  name       Name[]
  picture    Picture[]
  registered Registered[]
  Id         Id[]
}

model Location {
  id          Int          @id @default(autoincrement())
  street      String?      @db.VarChar(255)
  city        String?      @db.VarChar(255)
  state       String?      @db.VarChar(255)
  postcode    Int?
  userId      Int?
  user        User?        @relation(fields: [userId], references: [id])
  coordinates Coordinate[]
  timezone    Timezone[]
}
```

**Comment by austincrim** [comment, marked as answer]
Hey @NirinaTanjona 👋,

Your query should look like this:
```ts
let result = await prisma.user.findMany({
  where: {
    gender: 'female',
    location: {
      every: {
        city: 'Paris'
      }
    }
  }
})
```

Let me know if that works how you're intending!

**Reply by NirinaTanjona** [reply]
Hey 🖐🖐
The problem is that location is from another model. I want API to deal with get request like this
```http://localhost://3000/api/users?[param1]=[value1]&[param2]=[value2]&......```
Is something like this possible in prisma?

**Reply by austincrim** [reply]
When you say that "`Location` is from another model...", is it still in the same Prisma schema and same database? Prisma lets you write queries that touch data from multiple models, like my example showed above. 

Did you try using the query I provided? Did you run into any errors?

**Reply by vladi-stevanovic** [reply]
👋 Hello @NirinaTanjona  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
